src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
+            path = ProjectManager.getProjectRoot().fullPath;
+        }
+
+        return path;
+    }
+
     function _loadBowerJson(path) {
         _bowerJsonFile = new BowerJsonFile(path);
     }
 
+    function getDependencies() {
+        var deferred = new $.Deferred();
+
+        _bowerJsonFile.getContent()
+            .done(function (data) {
+                var dependencies = [];
+                data = JSON.parse(data);
+
+                _.each( data.dependencies, function (version, name) {
+                    dependencies.push({name:name, version:version});
+                } );
+
```

Comment:
My opinion on this one is that the filehandlers do all the things with the files like create, edit, delete, read, like in a datamodel.
The Manager will process the data retrieved from the file. This way there is more seperation of concerns and I think it will make it more clear